packages/zql/src/ivm/view-apply-change.ts

```diff
@@ -412,6 +417,19 @@ function getSingularEntry(parentEntry: Entry, relationship: string): RCEntry {
   return e as RCEntry;
 }
 
-function makeNewEntryWithRefCount(row: Row, rc: number): RCEntry {
-  return {...row, [refCountSymbol]: rc};
+function makeNewEntryWithRefCount(
+  row: Row,
+  schema: SourceSchema,
+  withIDs: boolean,
+  rc: number,
+): RCEntry {
+  const id = withIDs ? makeID(row, schema) : '';
+  return {...row, [refCountSymbol]: rc, [idSymbol]: id};
```

Comment:
@arv I know you were concerned with polymorphic object shape.  Does this look ok?  Would it be ok to make the [idSymbol] optional?

Reply:
I think we do not need to add the symbol if `withIDs` is `false`. In non SolidJS it will always be `false` and in SolidJS it will always be `true`.